cmd/tap/main.go

```diff
@@ -0,0 +1,278 @@
+package main
+
+import (
+	"context"
+	"log/slog"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	_ "github.com/joho/godotenv/autoload"
+
+	"github.com/carlmjohnson/versioninfo"
```

Comment:
we updated all projects to `github.com/earthboundkid/versioninfo/v2` (which is a drop-in maintained fork